Makefile

```diff
@@ -76,6 +76,7 @@ default: lib xxhsum_and_links
 all: lib xxhsum xxhsum_inlinedXXH
 
 xxhsum: xxhash.o xxhsum.o  ## generate command line interface (CLI)
+	$(CC) $(FLAGS) $^ $(LDFLAGS) -o $@$(EXT)
```

Comment:
minor : isn't this rule implied ?

Reply:
Not the FLAGS/CFLAGS part. Make defines it as the following:
```makefile
    $(CC) $(LDFLAGS) n.o $(LOADLIBES) $(LDLIBS)
```

Reply:
So, for example, if you did this:
```
make CFLAGS="-m32 -O3"
```
GCC would fail at the link step due to expecting 64-bit object files